spec/mailers/previews/user_mailer_preview_spec.rb

```diff
@@ -2,7 +2,7 @@
 require_relative './user_mailer_preview'
 
 RSpec.describe UserMailerPreview do
-  it_behaves_like 'a mailer preview'
+  it_behaves_like 'a mailer preview', preview_methods_that_can_be_missing: [:in_person_please_call]
```

Comment:
Is this meant to be temporary until we're fully moved over to the new name?

Reply:
yes